packages/epo-widget-lib/src/atomic/Blinker/Blinker.tsx

```diff
@@ -82,14 +82,10 @@ const Blinker: FunctionComponent<BlinkerProps> = ({
   useInterval(nextBlink, canBlink && loaded && playing ? interval : null);
 
   return (
-    <Styled.BlinkerContainer
-      className={className}
-      ref={ref}
-      isCondensed={isCondensed}
-    >
+    <Styled.BlinkerContainer className={className} ref={ref}>
       <Styled.BlinkerImages
         loadedCallback={() => setLoaded(true)}
-        {...{ images, activeIndex, isCondensed }}
+        {...{ images, activeIndex, $isCondensed: isCondensed }}
```

Comment:
most of the changes in here are for styled-components 6.0 renaming attributes to include the `$` prefix